dist/infragram.css

```diff
@@ -97,3 +97,8 @@ div#colorbar-container canvas#colorbar {
 #overlay-img {
   opacity:0.5;
 }
+
+ul li a:hover{
```

Comment:
@AbihaFatima can you target the `ul. dropdown-menu li:hover` selector just incase other `ul li` elements don't need a pointer cursor? Thanks!
```suggestion
ul.dropdown-menu li:hover {
```

Reply:
Done!